packages/xstate-wallet/src/store/memory-store.ts

```diff
@@ -167,12 +169,42 @@ export class MemoryStore implements Store {
     }
     channelEntry.setFunding(funding);
   }
-  public async getLedger(peerId: string) {
-    const ledgerId = this._ledgers[peerId];
 
-    if (!ledgerId) throw new Error(`No ledger exists with peer ${peerId}`);
+  public async lockLedger(participantId: string): Promise<LedgerLocked> {
```

Comment:
Usually the term used is `acquire` so maybe `acquireLockForLedgerChannel` is a better name?